packages/tupaia-web/src/utils/getFriendlyEntityType.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+
+export const getFriendlyEntityType = entityType => {
```

Comment:
Could you please add a comment here as it's hard to tell exactly why this needs to be done 🙏